test/components/groupTests.ts

```diff
@@ -1,210 +1,208 @@
 ///<reference path="../testReference.ts" />
 
 describe("ComponentGroups", () => {
-  it("append()", () => {
-    let componentGroup = new Plottable.Components.Group();
+  describe("Basic add/remove/has functionality", () => {
```

Comment:
I feel like that wouldn't work for root classes, like `Component`, `Dataset`, or `Plot`.